server/src/main/java/com/cloud/configuration/ConfigurationManagerImpl.java

```diff
@@ -1775,6 +1786,16 @@ private void checkZoneParameters(final String zoneName, final String dns1, final
         }
     }
 
+    private boolean isTungstenZone(long zoneId){
+        boolean isTungsten = false;
+        List<PhysicalNetworkVO> physicalNetworks = _physicalNetworkDao.listByZone(zoneId);
+        for(PhysicalNetworkVO physicalNetwork : physicalNetworks){
+            if(physicalNetwork.getIsolationMethods().contains("TF"))
+                isTungsten = true;
```

Comment:
@radu-todirica can 'return true' / 'break' here, instead of looping through other physical networks.

Reply:
Done.